wsi/drm/drmplane.cpp

```diff
@@ -252,6 +252,8 @@ bool DrmPlane::Initialize(uint32_t gpu_fd,
 
       formats_modifiers_.emplace_back(modifiers_obj);
     }
+
+    drmModeFreePropertyBlob(blob);
```

Comment:
Same comment as 'drmModeFreeProperty(broadcastrgb_props)' - this is not a NULL pointer dereference issue.  Was this reported by Klockwork?

Reply:
Klocwork caught this as a possible memory leak